### PR TITLE
Display content/_index.md if it exists

### PR DESCRIPTION
**Pull request by jamesericdavidson**
Improvement for #109. Test if `content/_index.md` exists, instead of configuring `enableFrontMatter`.

I thought the condition for displaying content on the home page could be made more convenient for people switching themes, or who want to add native Hugo front matter without theme configuration.

### CODE REVIEW

**Comment by yashmehrotra** [comment]
@jamesericdavidson The hugo build command is breaking, might want to make sure it works on your local first

**Comment by jamesericdavidson** [comment]
I think this is related to the fileExists function. Works on a local server, but GitLab CI/CD and GitHub build checks aren't happy. I'll have a look!

Regards,
James Davidson

-------- Original Message --------

**Comment by jamesericdavidson** [comment]
@yashmehrotra I committed a bad version of the file, but it's now fixed.

Additionally evaluating that `content/_index.md` isn't empty before creating an `<hr>` and empty `<section>`.

File: exampleSite/content/posts/theme-documentation-basics.md [hunk 1]
```diff
@@ -221,15 +221,6 @@ Text to display in the footer section
   footer = "Text in footer"
 ```
 
-### Displaying content on the homepage
-
-Content to display on homepage below the social icons, using the contents of `content/_index.md`.
-
```
Comment (yashmehrotra) [inline exampleSite/content/posts/theme-documentation-basics.md:227]: This should not be deleted, this documentation is still useful

Comment (jamesericdavidson) [reply exampleSite/content/posts/theme-documentation-basics.md]: Yes, you're right. I only wanted to remove the reference to `enableFrontMatter`.

This should've been an atomic commit as well, oops. Re-added.

**Comment by yashmehrotra** [comment]
Thanks, merging

**Comment by jamesericdavidson** [comment]
Thanks for merging

**Pull request merged**